.grenrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
```

Comment:
It seems we might benefit from quickly tagging recent commits so we can use grouping. Thoughts? https://github-tools.github.io/github-release-notes/options.html#group-by-1

Reply:
I tried, but usually we have up to 10 PRs per release, grouping creates a group per 1-2 PRs

Reply:
We can always revisit this later too.